Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -168,6 +168,70 @@
   - type: TypingIndicator
     proto: robot
 
+- type: entity
+  parent: MobSiliconBase
+  id: MobMuleBot
+  name: mulebot
+  description: Delivers cargo!
+  components:
+  - type: NoSlip
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Silicon/Bots/mulebot.rsi
+    layers:
+    - state: mulebot
+      map: ["enum.VehicleVisualLayers.AutoAnimate"]
+    noRot: true
+  - type: IntrinsicRadioReceiver
+  - type: ActiveRadio
+    channels:
+    - Common
+  - type: GhostRole
+    makeSentient: true
+    name: ghost-role-information-mulebot-name
+    description: ghost-role-information-mulebot-description
+  - type: GhostTakeoverAvailable
+  - type: MovementSpeedModifier
+    weightlessModifier: 0
+    acceleration: 2
+    friction: 2
+    frictionNoInput: 6
+    baseWalkSpeed: 3
+    baseSprintSpeed: 6
+  - type: AmbientSound
```

Comment:
i think it shouldn't always produce sounds
imagine someone who playing on mulebot and listening that engine sound all the time